fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -35,37 +35,43 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
         violation is at most :code:`2*(eps+best_gap)`
     :type eps: float
 
-    :param T: Maximum number of iterations
-    :type T: int
+    :param max_iter: Maximum number of iterations
+    :type max_iter: int
 
     :param nu: Convergence threshold for the duality gap, corresponding to a
         conservative automatic setting based on the statistical uncertainty in measuring
         classification error
     :type nu: float
 
-    :param eta_mul: Initial setting of the learning rate
-    :type eta_mul: float
+    :param eta0: Initial setting of the learning rate
+    :type eta0: float
+
+    :param run_linprog_step: if True each step of exponentiated gradient is followed by the saddle
+        point optimization over the convex hull of classifiers returned so far; default True
+    :type run_linprog_step: bool
     """
```

Comment:
i'm being a bit nitpicky. but can we just do `linprog_step` rather than `run_linprog_step`?

Reply:
That doesn't soudn like a bool, though.

Reply:
sure